cli/commands/seek.py

```diff
@@ -6,33 +6,37 @@
 
 
 @click.command(options_metavar='[<options>]')
+@click.option(
+    '-f', '--forward', is_flag=True,
+    help='Increment playback position'
+)
+@click.option(
+    '-r', '--rewind', is_flag=True,
```

Comment:
i think this would be more intuitive if named as "reverse" instead of "rewind"? docstring should be updated as well to reflect that